docs/guide/data_types/timeseries.rst

```diff
@@ -250,22 +250,27 @@ type:
 
     import sunpy.timeseries as ts
     import sunpy.data.sample
-    ts_plot = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS')
-    fig = ts_plot.plot()
+    ts = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS')
+    ts.peek()
 
-This will open a matplotlib plot on your screen. The `~sunpy.timeseries.GenericTimeSeries.peek`
+This will open a Matplotlib plot on your screen. The `~sunpy.timeseries.GenericTimeSeries.peek`
 method provides a view on data customised for each source while `~sunpy.timeseries.GenericTimeSeries.plot`
 provides a more general plot.  Note that `~sunpy.timeseries.GenericTimeSeries.peek`
 returns a `matplotlib.figure.Figure` object, if you want to save this to a PNG
```

Comment:
I thought it returned `None`?

Reply:
It does, but it also shows the figure.

Reply:
Oh woops, just saw the line you were commenting on...